<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Smoothing</class>
 <widget class="QDialog" name="Smoothing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>平滑</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>平滑方法：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="valueSigmaC">
     <property name="text">
      <string>0.1 </string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="valueSigma">
     <property name="text">
      <string>0.1 </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelSigmaC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;σ&lt;span style=&quot; vertical-align:sub;&quot;&gt;C&lt;/span&gt;：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>均值滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中值滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>最大值滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>最小值滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>高斯滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>双边滤波</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelSigma">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>σ：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelSigmaS">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;σ&lt;span style=&quot; vertical-align:sub;&quot;&gt;S&lt;/span&gt;：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="sliderSigma">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelR">
     <property name="text">
      <string>半径大小：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="sliderSigmaS">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="valueSigmaS">
     <property name="text">
      <string>0.1 </string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="valueR">
     <property name="text">
      <string>1   </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="sliderR">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="sliderSigmaC">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
